### PR TITLE
Update README.en.md

### PR DESCRIPTION
**Pull request by VulcanSphere**
Synchronising with latest Indonesian language version

### CODE REVIEW

File: README.en.md [hunk 1]
```diff
@@ -1,16 +1,26 @@
...
+    <a href="#"><img src="https://img.shields.io/static/v1?label=year%20released:&message=2019&color=yellowgreen&style=plastic%22"></a>
+    <a href="https://github.com/bebasid/bebasid/actions?query=workflow%3AValidate"><img src="https://img.shields.io/github/actions/workflow/status/bebasid/bebasid/validate.yml?branch=master&logo=github&logoColor=fff&label=validation" alt="Hosts status"></a>
+    <img src="https://img.shields.io/github/size/bebasid/bebasid/releases/hosts.svg?label=size" alt="Release size">
+    <a href="#"><img src="https://hits.seeyoufarm.com/api/count/incr/badge.svg?url=https%3A%2F%2Fgithub.com%2Fbebasid%2Fbebasid_fGHyh&count_bg=%234572CD&title_bg=%23555555&icon_color=%23E7E7E7&title=watcher%3A+%28today%2Ftotal%29&edge_flat=false"/></a>
+    <a href="https://github.com/bebasid/bebasid/blob/master/LICENSE"><img src="https://img.shields.io/badge/license-MIT-green"></a>
 </p>
 <p align="center">
-    <a href="README.md">Bacakan README dalam Bahasa Indonesia</a>
+    <a href="README.md">Indonesia</a> | <b>English</b> (this one)
```
Comment (nxzlem) [inline README.en.md:23]: (this one) should be removed in this section.

File: README.en.md [hunk 2]
```diff
@@ -1,16 +1,26 @@
...
-</p>
+<a href="https://discord.gg/EKrxZyu"><img src="https://discordapp.com/api/guilds/630415907021389825/widget.png?style=banner2" alt="Join BEBASID Discord"></a>
+ </p>
 <p align="center">
-    <a href="https://discord.gg/EKrxZyu"><img src="https://discordapp.com/api/guilds/630415907021389825/widget.png?style=banner4" alt="Discord Banner 4"/></a>
+    <a href="#"><img src="https://img.shields.io/static/v1?label=year%20released:&message=2019&color=yellowgreen&style=plastic%22"></a>
+    <a href="https://github.com/bebasid/bebasid/actions?query=workflow%3AValidate"><img src="https://img.shields.io/github/actions/workflow/status/bebasid/bebasid/validate.yml?branch=master&logo=github&logoColor=fff&label=validation" alt="Hosts status"></a>
+    <img src="https://img.shields.io/github/size/bebasid/bebasid/releases/hosts.svg?label=size" alt="Release size">
+    <a href="#"><img src="https://hits.seeyoufarm.com/api/count/incr/badge.svg?url=https%3A%2F%2Fgithub.com%2Fbebasid%2Fbebasid_fGHyh&count_bg=%234572CD&title_bg=%23555555&icon_color=%23E7E7E7&title=watcher%3A+%28today%2Ftotal%29&edge_flat=false"/></a>
```
Comment (nxzlem) [inline README.en.md:19]: `watcher`, should be `watchers`

File: README.en.md [hunk 3]
```diff
@@ -1,16 +1,26 @@
+<p align="center">
+    <b>For ISP that implemented DPI, use <a href="https://github.com/bebasid/bebasit">bebasit</a> to bypass the DPI or turn on IPv6 if the ISP support it.</b>
```
Comment (nxzlem) [inline README.en.md:2]: `implemented` should be `uses` and for the turn on ipv6, add s at the end of the word `support`

File: README.en.md [hunk 4]
```diff
@@ -1,16 +1,26 @@
...
+    <br><sup>Signs if your ISP implemented DPI: HTTP redirected to<code><a href="http://lamanlabuh.aduankonten.id" target="_blank">lamanlabuh.aduankonten.id</a></code> or HTTPS responded with <code>ERR_CONNECTION_RESET</code>/<code>PR_CONNECT_RESET_ERROR</code></sup><br>
+    <b>Bypass MCI DPI on Modem/Router (in Indonesian):</b><br>
```
Comment (nxzlem) [inline README.en.md:4]: fix this typo issue on the MCI word section

File: README.en.md [hunk 5]
```diff
@@ -1,16 +1,26 @@
+<p align="center">
+    <b>For ISP that implemented DPI, use <a href="https://github.com/bebasid/bebasit">bebasit</a> to bypass the DPI or turn on IPv6 if the ISP support it.</b>
+    <br><sup>Signs if your ISP implemented DPI: HTTP redirected to<code><a href="http://lamanlabuh.aduankonten.id" target="_blank">lamanlabuh.aduankonten.id</a></code> or HTTPS responded with <code>ERR_CONNECTION_RESET</code>/<code>PR_CONNECT_RESET_ERROR</code></sup><br>
```
[the remark this answers is not in the document]
Comment (VulcanSphere) [reply README.en.md:3]: Done

**Review by nxzlem — APPROVED** [review]
Approved

**Pull request merged**